CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Fixes
 
+- Update the development environment for docker-compose v2.
```

Comment:
OIC.  I misunderstood the format. I thought 2.1.1 was the Upcoming version. 🤦‍♂️